ihcsdk/ihcclient.py

```diff
@@ -483,20 +493,38 @@ def wait_for_resource_value_changes(self, wait: int = 10):
         )
         if xdoc is False:
             return False
+
+        # Uncomment to print entire changes element for reverse engineering
+        #        element_string = ElementTree.tostring(xdoc, encoding="unicode")    
+        #        print (element_string)            
```

Comment:
Maybe a "debug" flag in a class field would be better than than outcommented code.